pkg/exec/exec.go

```diff
@@ -82,3 +82,40 @@ func startReaderGoroutine(reader io.Reader, show bool, cmdOutput *string, consol
 	return result
 
 }
+
+// CreateConsoleOutputWriterAndChannel is a utility function that returns a pipeWriter and a channel;
+// any strings written to that PipeWriter will be output to the channel (as lines) when the
+// writer closes. This is used to retrieve the stdout/stderr output from the container exec commands.
+//
+// The io.PipeWriter can be passed to ExecuteCommand(...) above, in order to receive the full
+// stderr/stdout output from the process.
+// See calling functions of CreateConsoleOutputWriterAndChannel for examples of usage.
+func CreateConsoleOutputWriterAndChannel() (*io.PipeWriter, chan []string) {
```

Comment:
Sorry but we should have a unit test for this command :(

Reply:
Done.